src/components/tabs/js/tabsController.js

```diff
@@ -163,6 +163,9 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
 
   function handleMaxTabWidth (newWidth, oldWidth) {
     if (newWidth !== oldWidth) {
+      angular.forEach(elements.tabs, function(tab) {
+        tab.style.maxWidth = newWidth + 'px';
```

Comment:
It does, handleMaxTabWidth is the setter function for the maxTabWidth property on the [ctrl](https://github.com/Emeegeemee/material/blob/619a7a3adffa05e1340157792ee0064cf1b74ac7/src/components/tabs/js/tabsController.js#L25). It does need to handle the max width changing or else the tabs content get cut off, from my manual testing.